lib/OpenQA/Worker/Jobs.pm

```diff
@@ -914,8 +902,10 @@ sub check_backend {
     log_debug("checking backend state");
 
     return log_debug("backend is running") if $worker->{child}->is_running();
+    return log_debug("backend state not known") unless defined $worker->{child}->exit_status;
```

Comment:
I would move this to log_error

Reply:
actually we could remove that timer now, if stop_job would not be so 'blocking' - give me some time to try to re-elaborate this